src/robot/libraries/String.py

```diff
@@ -261,6 +261,41 @@ def get_lines_matching_regexp(self, string, pattern):
         regexp = re.compile('^%s$' % pattern)
         return self._get_matching_lines(string, regexp.match)
 
+    def get_regexp_matches(self, string, pattern, *groups):
+        """Returns multiple match results of the given ``string`` that match the ``pattern``.
+        This keyword returns all matches(don't return on first match) and support group.
```

Comment:
Should have an empty line after the first docstring line. For details see https://www.python.org/dev/peps/pep-0257/.